pyiron_base/project/path.py

```diff
@@ -391,10 +391,7 @@ def _create_path(self, path, rel_path=None):
         if rel_path:
             rel_path = self._windows_path_to_unix_path(rel_path)
             path = posixpath.join(path, rel_path)
-        try:
```

Comment:
The issue was this part - rather than using `os.makedirs(exist_ok=True)` we used our own `try except` which ignored the error when the folder was not created.